### ISSUE TITLE
Slack notifications after title include "\n" in printed text rather than rendered new line

### ISSUE THREAD

**Issue by MylesGray** [lokenx/plexrequests-meteor#344]
As such:

```
PlexRequests Bot BOT [16:10] 
Plex Requests tv:\nSuits requested by asdfgh
```

Line 22 in sendNotification.js seems to be at fault:

```
if (settings.slackENABLED) {
        Meteor.call("sendSlackNotification", settings, message.title + ":\\n" + message.body)
      }
```

**Comment by rigrassm** [comment]
Not gonna lie, I thought encoding was handled by the package used to make the HTTP call. I don't use slack but I'll change it to manually encode the message before it's passed to the call and see if that fixes it.

**Comment by rigrassm** [comment]
Fixed with release v1.11.2.
